src/api/authorData.js

```diff
@@ -38,18 +38,18 @@ const getSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 });
 
 // FIXME: DELETE AUTHOR
-const deleteSingleAuthor = (firebaseKey, uid) => new Promise((resolve, reject) => {
+const deleteSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
   axios.delete(`${dbUrl}/authors/${firebaseKey}.json`)
     .then(() => {
-      getAuthors(uid).then((authorsArray) => resolve(authorsArray));
+      getAuthors(firebaseKey.uid).then((authorsArray) => resolve(authorsArray));
```

Comment:
```suggestion
getAuthors(uid).then((authorsArray) => resolve(authorsArray));
```
firebaseKey is a a key on an object so you will not be able to use dot notation.